torch/csrc/Generator.h

```diff
@@ -13,6 +14,7 @@ struct THPGenerator {
   ((THPGenerator*)obj)->cdata
 
 THP_API PyObject * THPGenerator_New();
+THP_API PyObject * THPGenerator_NewWithGenerator(THGenerator *cdata);
```

Comment:
Can you document that generator is not stolen and the caller should ensure that it’s alive for longer than the returned object?